test/util/Fixtures.scala

```diff
@@ -243,9 +246,20 @@ trait BaseSpec extends UnitSpec with OneAppPerSuite with PatienceConfiguration w
 
   implicit val hc = HeaderCarrier()
 
+  val fakeConfig = ConfigFactory.parseMap(
```

Comment:
Change this to be overriden within the GuiceApplicationBuilder. Remove the binding for Config!